examples/clearsign.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
+@include_once dirname(__FILE__).'/../vendor/autoload.php';
```

Comment:
Why don't you use the `require_once` because of consistency.

And remove `@` to avoid inhibiting the error.

Reply:
If adding the `vendor/autoload.php` file successfully, the following required files seem that they're unnecessary.

Perhaps we can let developers use these sample codes should use `composer` to install the dependencies correctly.

Reply:
if you read the discussion, it's because this library is used by some who do not use composer and so the autoload file should not be required.

Reply:
@singpolyma, thank you for your reply.

I see that. It's fine to add this line to let developers use these sample codes without using `composer`.